src/modules/launcher/PowerLauncher/LauncherControl.xaml

```diff
@@ -38,11 +38,21 @@
                                     </Style>
                                 </TextBlock.Style>
                             </TextBlock>
-                            <Border x:Name="border" Background="{TemplateBinding Background}" BorderThickness="{TemplateBinding BorderThickness}" BorderBrush="{TemplateBinding BorderBrush}" SnapsToDevicePixels="True">
-                                <ScrollViewer x:Name="PART_ContentHost" Background="{TemplateBinding Background}" Focusable="false" HorizontalScrollBarVisibility="Hidden" VerticalScrollBarVisibility="Hidden">
+                            <Border x:Name="border"
+                                    Background="{TemplateBinding Background}"
+                                    BorderThickness="{TemplateBinding BorderThickness}"
+                                    BorderBrush="{TemplateBinding BorderBrush}"
```

Comment:
```suggestion
                                    BorderBrush="{TemplateBinding DialogBorderBrush}"
```

Reply:
Thanks @htcfreek!

Reply:
Reverting this back. It should be TemplateBinding BorderBrush, using DialogBorderBrush here caused the crash.

Reply:
That's interesting. Didn't find "BorderBrush" in the Theme definition. Thought you replaced "BorderBrush" with "DialogBorderBrush". 🤔

Reply:
That's interesting. Didn't find "BorderBrush" in the theme definition files. Thought you replaced "BorderBrush" with "DialogBorderBrush". 🤔

Reply:
We are TemplateBinding here, meaning that we are binding to the color set on the control itself. Not a resource that is defined in a resourcedictionary.

Reply:
💡